lib/yelp/responses/models/rating.rb

```diff
@@ -3,8 +3,35 @@
 module Yelp
   module Response
     module Model
-      class Rating < Response::Base
-        attr_reader :rating, :img_url
+      class Rating
+        attr_reader :rating
+
+        def initialize(json)
+          @rating = json['rating']
+          @image  = { regular: json['rating_image_url']       || json['rating_img_url'],
+                      small:   json['rating_image_small_url'] || json['rating_img_url_small'],
+                      large:   json['rating_image_large_url'] || json['rating_img_url_large'] }
+        end
+
+        def image_url(size = :regular)
+          @image[size] if @image[size]
+        end
+
+        def image_small_url
+          image_url(:small)
+        end
+
+        def image_large_url
+          image_url(:large)
+        end
+
+        # The naming from the response is consistent (img vs image, url_small vs small_url),
+        # the aliases try to "standardize" them for the end user
+        alias_method :img_url, :image_url
```

Comment:
This doesn't seem like standardizing to me. If we handle the different response names in the parsing, I think it'd be better if the clientlib stuck to one or the other instead of offering both.